packages/office-ui-fabric-react/src/components/Slider/Slider.scss

```diff
@@ -1,24 +1,143 @@
 @import '../../common/common';
 
-.root {
-  user-select: none;
-  margin-bottom: 8px;
-}
+.row {
```

Comment:
Can you avoid this nesting? Every rule just became 2 levels of specificity which would break a consumer's CSS which may have been more specific than these rules.